cw_monero/lib/api/transaction_history.dart

```diff
@@ -27,7 +33,6 @@ List<Transaction> getAllTransactions() {
   List<Transaction> dummyTxs = [];
 
   txhistory ??= monero.Wallet_history(wptr!);
-  monero.TransactionHistory_refresh(txhistory!);
```

Comment:
To not call it multiple times, it will be called only in one function with a mutex to prevent the crash that occurs when the wallet is large, and refresh is happening on a separate thread